config/jobs/cert-manager/cert-manager/release-next/cert-manager-release-next-periodics.yaml

```diff
@@ -1,505 +1,8 @@
-    preset-bazel-scratch-dir: "true"
-    preset-venafi-cloud-credentials: "true"
-    preset-venafi-tpp-credentials: "true"
-    preset-ginkgo-focus-venafi: "true"
-  spec:
-    containers:
-    - image: eu.gcr.io/jetstack-build-infra-images/bazelbuild:20210831-4cf7b0b-4.2.1
-      args:
-      - runner
-      - devel/ci-run-e2e.sh
-      resources:
-        requests:
-          cpu: 3500m
-          memory: 12Gi
-      env:
-      - name: K8S_VERSION
-        value: "1.23"
-      securityContext:
-        privileged: true
-        capabilities:
-          add: ["SYS_ADMIN"]
-      volumeMounts:
-      - mountPath: /lib/modules
-        name: modules
-        readOnly: true
-      - mountPath: /sys/fs/cgroup
-        name: cgroup
-    volumes:
-    - name: modules
-      hostPath:
-        path: /lib/modules
-        type: Directory
-    - name: cgroup
-      hostPath:
-        path: /sys/fs/cgroup
-        type: Directory
-    dnsConfig:
-      options:
-      - name: ndots
-        value: "1"
-
-- name: ci-cert-manager-next-upgrade
-  interval: 168h # 1 week
-  agent: kubernetes
-  decorate: true
-  # extra refs specify what repo should be cloned
-  extra_refs:
-  - org: cert-manager
-    repo: cert-manager
-    base_ref: release-1.8
-  annotations:
-    testgrid-create-test-group: 'true'
-    testgrid-dashboards: jetstack-cert-manager-master
-    testgrid-alert-email: cert-manager-dev-alerts@googlegroups.com
-    description: Runs cert-manager upgrade test.
-  labels:
-    preset-service-account: "true"
-    preset-dind-enabled: "true"
-    preset-bazel-remote-cache-enabled: "true"
-    preset-bazel-scratch-dir: "true"
-  spec:
-    containers:
-    - image: eu.gcr.io/jetstack-build-infra-images/bazelbuild:20210831-4cf7b0b-4.2.1
-      args:
-      - runner
-      - make
-      - cluster
-      - verify_upgrade
-      resources:
-        requests:
-          cpu: 3500m
-          memory: 12Gi
-      env:
-      - name: K8S_VERSION
-        value: "1.23"
-      securityContext:
-        privileged: true
-        capabilities:
-          add: ["SYS_ADMIN"]
-      volumeMounts:
-      - mountPath: /lib/modules
-        name: modules
-        readOnly: true
-      - mountPath: /sys/fs/cgroup
-        name: cgroup
-    volumes:
-    - name: modules
-      hostPath:
-        path: /lib/modules
-        type: Directory
-    - name: cgroup
-      hostPath:
-        path: /sys/fs/cgroup
-        type: Directory
-    dnsConfig:
-      options:
-      - name: ndots
-        value: "1"
+# We don't need periodically testing the release-next breanch (e.g., the
+# "release-1.9" branch) until we release the first alpha (e.g.,
+# "1.9.0-alpha.0"). Since we can't "deactivate" the release-next periodics
+# jobs until we have an alpha (there is no "skip" field on the ProwJob
+# object), we just remove all of the jobs from this file. That's why the
+# below array is empty. See Step 13.3 in
+# https://cert-manager.io/docs/contributing/release-process/
```

Comment:
nit(non-blocking): grammar
```suggestion
# We don't need to periodically test the release-next breanch (e.g., the
# "release-1.9" branch) until we release the first alpha (e.g.,
# "1.9.0-alpha.0"). Since we can't "deactivate" the release-next periodics
# jobs until we have an alpha (there is no "skip" field on the ProwJob
# object), we just remove all of the jobs from this file. That's why the
# below array is empty. See Step 13.3 in
# https://cert-manager.io/docs/contributing/release-process/
```

Reply:
It's all good, I'm fixing it in the PR I'm working on for release-previous :+1: